src/main/java/hudson/tasks/junit/JUnitTask.java

```diff
@@ -16,4 +16,7 @@ public interface JUnitTask {
     boolean isSkipPublishingChecks();
 
     String getChecksName();
+
+    boolean isParseOldReports();
```

Comment:
```suggestion
    default boolean isParseOldReports() {
       return true;
    }
```
https://github.com/jenkinsci/uipath-automation-package-plugin/blob/b391e63b45a924782c36b12de4047641e3d910c8/src/main/java/com/uipath/uipathpackage/UiPathTest.java#L44 whatever that is